src/vs/workbench/contrib/inlineChat/common/inlineChat.ts

```diff
@@ -196,3 +196,14 @@ Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfigurat
 		}
 	}
 });
+
+Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfiguration({
```

Comment:
You can merge this `registerConfig`-call into the one above, simply merge the properties